<launch>
	<include file="$(find mrover_arm)/launch/demo.launch">
    </include>
	
	<node name="mrover_arm_follow_joint_trajectory" pkg="mrover" type="arm_trajectory"></node>
</launch>